test/strategies/aave/tests.js

```diff
@@ -0,0 +1,1443 @@
+const hre = require('hardhat');
+const chai = require('chai');
+
+const { expect } = require('chai');
+chai.use(require('chai-as-promised'));
+
+const { configure } = require('@defisaver/sdk');
+const {
+    assets,
+    getAssetInfo,
+    utils: { compare },
+} = require('@defisaver/tokens');
+
+const {
+    getAddrFromRegistry,
+    getProxy,
+    redeploy,
+    redeployCore,
+    setBalance,
+    openStrategyAndBundleStorage,
+    fetchAmountinUSDPrice,
+    resetForkToBlock,
+    network,
+    addrs,
+    chainIds,
+    Float2BN,
+    nullAddress,
+    setNewExchangeWrapper,
+    getLocalTokenPrice,
+    balanceOf,
+    BN2Float,
+    takeSnapshot,
+    revertToSnapshot,
+    ETH_ADDR,
+} = require('../../utils');
+
+const {
+    addBotCaller,
+    createStrategy,
+    createBundle,
+    activateSub,
+} = require('../../utils-strategies');
+
+const {
+    createAaveV3CloseToCollWithMaximumGasPriceStrategy,
+    createAaveV3FLCloseToCollWithMaximumGasPriceStrategy,
+    createAaveV3CloseToDebtWithMaximumGasPriceStrategy,
+    createAaveV3FLCloseToDebtWithMaximumGasPriceStrategy,
+} = require('../../strategies');
+
+const {
+    aaveV3Supply,
+    aaveV3Borrow,
+} = require('../../actions');
+
+const { RATIO_STATE_OVER } = require('../../triggers');
+const { subAaveV3CloseWithMaximumGasPriceBundle } = require('../../strategy-subs');
+const {
+    callAaveCloseToCollWithMaximumGasPriceStrategy,
+    callAaveFLCloseToCollWithMaximumGasPriceStrategy,
+    callAaveCloseToDebtWithMaximumGasPriceStrategy,
+    callAaveFLCloseToDebtWithMaximumGasPriceStrategy,
+} = require('../../strategy-calls');
+const { subAaveV3CloseBundle } = require('../../l2-strategy-subs');
+const { callAaveFLCloseToDebtL2Strategy } = require('../../l2-strategy-calls');
+
+const testPairs = [
+    {
+        collAsset: 'WETH',
+        debtAsset: 'DAI',
+    },
+    {
+        collAsset: 'WBTC',
+        debtAsset: 'USDC',
+    },
+    {
+        collAsset: 'DAI',
+        debtAsset: 'WETH',
+    },
+];
+
+const deployCloseToCollWithMaximumGasPriceBundle = async (proxy, isFork = undefined) => {
```

Comment:
`undefined` is a falsy value in js, but to keep things more clear assign `isFork = false` to be more clear